packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md

```diff
@@ -308,6 +307,56 @@ function error(err) {
 }
 ```
 
+
+## Using with Okta's SDKs
+
+Okta provides a number of SDKs that you might want to use the Sign-In Widget with, including Angular, React, and Vue.
+
+Using the Sign-In Widget with our SDKs that target the web is fairly straightforward.
+
+### Angular
+
+The [Okta Sign-In Widget and Angular guide](/code/angular/okta_angular_sign-in_widget/) shows the code you'll need to embed the Sign-In Widget in an Angular app.
+
+See the [Okta Angular + Custom Login Example](https://github.com/okta/samples-js-angular/tree/master/custom-login) for a working example.
+
+### React
+
+The [Okta Sign-In Widget and React guide](/code/react/okta_react_sign-in_widget/) shows the code you'll need to embed the Sign-In Widget in a React app.
+
+See the [Okta React + Custom Login Example](https://github.com/okta/samples-js-react/tree/master/custom-login) for a working example.
+
+### Vue
+
+The [Okta Sign-In Widget and Vue guide](/code/vue/okta_vue_sign-in_widget/) shows the code you'll need to embed the Sign-In Widget in a Vue app.
+
+^^ todo: <https://github.com/okta/okta-developer-docs/issues/1440>
+
+See the [Okta Vue + Custom Login Example](https://github.com/okta/samples-js-vue/tree/master/custom-login) for a working example.
+
+### Mobile SDKs
+
+We also have mobile SDKs for Android, React Native, iOS, and Xamarin.
+
+For mobile apps, we've found it's difficult (or impossible) to embed the Sign-In Widget. If you redirect to Okta for authentication, you can [customize the hosted sign-in widget](/docs/guides/custom-hosted-signin/overview/). However, if you want to have a custom login form and have that provided by the Sign-In Widget, you're (currently) out of luck.
+
+The good news is our mobile SDKs provide support for you to build your own UI.
+
+- Android:
+    - [Sign in with your own UI](https://github.com/okta/okta-oidc-android#Sign-in-with-your-own-UI)
+    - [Custom Sign In Example](https://github.com/okta/samples-android/tree/master/custom-sign-in)
+- iOS:
+    - [Authenticate a User](https://github.com/okta/okta-auth-swift#authenticate-a-user)
+    - [Okta iOS Custom Sign In Example](https://github.com/okta/samples-ios/tree/master/custom-sign-in)
+- React Native
+    - todo: okta-react-native has no docs like Android does - <https://github.com/okta/okta-react-native/issues/7>
+    - [Okta React Native Custom Sign In Example](https://github.com/okta/samples-js-react-native/tree/master/custom-sign-in)
+- Xamarin
+    - todo: link to code section of Xamarin SDK README that shows code for custom UI
+    - todo: link to custom sign in example - <https://github.com/okta/okta-oidc-xamarin/issues/15>
+
```

Comment:
@Quorralyne Do we have a Blazor SDK? If so, is it mature enough to list here? I'm thinking of changing this to just add a link to Xamarin since it's still in Alpha.